### ISSUE TITLE
Could not load translate provider "gct"

### ISSUE THREAD

**Issue by KindCoder-no** [shawnvogt/strapi-provider-translate-gct#2]
I get this error when trying to run the strapi application. I have added apikey

![Skjermbilde 2023-04-26 kl  12 15 12](https://user-images.githubusercontent.com/40148297/234545376-c4be25dd-d6a4-42a6-9094-0b24eae1a07b.png)

**Comment by shawnvogt** [comment]
Hi @KindCoder-no, 

I had no idea anyone other than myself and the company I work for were using this plugin. To be honest, when I set this up I did kind of the bare minimum to get a gct provider set up for [strapi-plugin-translate](https://github.com/fekide/strapi-plugin-translate) since I was in a rush at the time and I haven't really looked at this repo since then. 

I will be back on the project where we are using this plugin within a couple of weeks and I do plan to clean it up and ensure everything is playing nice with any updates that have happened with strapi-plugin-translate since I last looked at it and I will be in a much better position to assist with issues here at that point.

If you find anything in the code causing issues for you and you have a fix for it, I welcome your pr.

My intention was for this project to be temporary until I find time to include the plugin in the strapi-plugin-translate project. If you are up to the task, feel free to start that process, I'm sure the contributors over at strapi-plugin-translate would love another contributor and a gct plugin in their repo. 

Sorry, I can't be of more immediate help.
